velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -103,3 +120,95 @@ TEST_F(ParquetReaderTest, parseRowMapArray) {
   auto col0_1_1_0 = col0_1_1->childAt(0);
   EXPECT_EQ(col0_1_1_0->type->kind(), TypeKind::INTEGER);
 }
+
+TEST_F(ParquetReaderTest, varcharFilters) {
+  // name < 'CANADA'
+  FilterMap filters;
```

Comment:
The re-use of this variable is confusing. It looks like the test is appending to the map, but it doesn't because std::move clears the map. Would it be clearer to pass the map to assertReadWithFilters explicitly?

`assertReadWithFilters(
      "nation.parquet", rowType, std::move(filters), {"name", exec::lessThan("CANADA")});`